### DISCUSSION TITLE
Nouvelle Box Home-Assistant Green - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#24] [Comments]
# Nouvelle Box Home-Assistant Green - Haade.fr

10 ans déjà que home-assistant existe, pour cet événement ils nous font découvrir leur nouvelle box Home-assistant green à moins de 100€

https://haade.fr/fr/blog/nouvelle-box-home-assistant-green-anniversaire-10-ans

**Comment by fredo3333** [comment]
Bonjour, homeassistant green ne gère pas le bluetooth, il faut rajouter une clé USB. En conséquence, les 2 ports sont occupés par skyconnect et le bluetooth.
Pour le prix, il ne comprend pas les taxes...
